packages/jest-cli/src/__tests__/watch_filename_pattern_mode.test.js

```diff
@@ -135,7 +135,7 @@ describe('Watch mode flows', () => {
 
     // globalConfig is updated with the current pattern
     expect(runJestMock.mock.calls[0][0].globalConfig).toEqual({
-      onlyChanged: true,
```

Comment:
this should be the correct value. If we have a pattern, we don't match only changed files, we match all files